File: 1주차/PGS_최소직사각형/seoheun.py
def solution(sizes):
    answer = 0
    big_x = 0
    big_y = 0
    
    for size in sizes:
        if size[0] > size[1]:
            temp = size[0]
            size[0] = size[1]
            size[1] = temp
    
    for size in sizes:
        if big_x < size[0]:
            big_x = size[0]
        if big_y < size[1]:
            big_y = size[1]
        
    answer = big_x * big_y
    
    return answer

File: 4주차/PGS_카펫/seoheun.js
function solution(brown, yellow) {
    var answer = [];

    let pair = [];

    // 약수
    for (let i = 1; i <= yellow; i++) {
        if (yellow % i === 0) {
            pair.push(i);
        }
    }

    let total = brown + yellow;

    for (let i = 0; i < pair.length; i++) {
        let temp = yellow / pair[i];
        if (pair[i] * 2 + temp * 2 + 4 === brown) {
            answer = [pair[i] + 2, temp + 2];
            return answer.reverse();
        }
    }
}
